type_classes.md

```diff
@@ -92,7 +92,7 @@ We can now reimplement `double` using an instance implicit by:
 # instance : Add Nat where
 #  add := Nat.add
 # instance : Add Int where
-#  add := Int.add
+#  add := Int.mul
```

Comment:
These lines starting with `#` are hidden in the rendered book, and I assume they generally match code already shown earlier in the chapter. In the code block just above this one, this instance is defined to use `Int.add`, so readers would expect that definition here as well.
https://github.com/leanprover/theorem_proving_in_lean4/blob/6643a2796f29459742b73c044600710f71b0b824/type_classes.md?plain=1#L76-L77

I think what actually happened here is a copy-paste error from
```lean
#eval double { add := Nat.add } 10
-- 20

#eval double { add := Nat.mul } 10
-- 100

#eval double { add := Int.add } 10
-- 20
```
and since I don't see an easy way to keep the `Nat.mul` example without confusing the reader, I would recommend just changing `100` to `20`